File: 9. Palindrome Number/palindromeNumber.py
class Solution(object):
    def isPalindrome(self, x):
        if x < 0:
            return False
        if x == 0:
            return True
        m = 0
        t = x
        while x:
            m = m*10 + x%10
            x = x/10
        if t == m:
            return True
        else:
            return False

File: 9. Palindrome Number/palindromeNumber.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        if(x < 0)
            return false;
            
        if(x == 0)
            return true;
        int m = 0, t =x;
        
        while(x){
            m = m * 10 + x % 10;
            x/=10;
        }
        return t==m;

    }
};